terraform-beyond-layer0/example-3/env-dev/main.tf

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
I see what you mean - given this is more "simple" terraform config, it kind of makes more sense to me as example 2. Then, we compact everything down in example 3. 

To add to the value of this example, I think we should add add some sections talking about how to "release" your services as modules. I'd be open to pair on it.